server/src/main/kotlin/org/itmodreamteam/myrest/server/service/table/TableWaiterService.kt

```diff
@@ -0,0 +1,15 @@
+package org.itmodreamteam.myrest.server.service.table
+
+import org.itmodreamteam.myrest.shared.restaurant.EmployeeInfo
+import org.itmodreamteam.myrest.shared.table.TableView
+
+interface TableWaiterService {
+
+    fun addWaiter(tableId: Long, waiterId: Long): EmployeeInfo
+
+    fun removeWaiter(tableId: Long, waiterId: Long)
+
+    fun getTableWaiters(tableId: Long): List<EmployeeInfo>
```

Comment:
It's common to have one waiter per table

Reply:
Might be, idk, but we are flexible!